src/input.rs

```diff
@@ -701,8 +702,8 @@ mod tests {
                     mouse_bindings: &config.mouse_bindings()[..],
                 };
 
-                if let Event::WindowEvent { event: WindowEvent::MouseInput { state, button, .. }, .. } = $input {
-                    processor.mouse_input(state, button);
+                if let Event::WindowEvent { event: WindowEvent::MouseInput { state, button, modifiers, .. }, .. } = $input {
+                    processor.mouse_input(state, button, modifiers.shift);
```

Comment:
It seems to me that `modifiers` should be passed through in its entirety. If we ever want to add some other mouse + modifier functionality, then all the mods are already being passed without bloating the method signature.